public/app/features/plugins/sql/components/QueryHeader.tsx

```diff
@@ -48,6 +49,11 @@ export function QueryHeader({
 
   const onEditorModeChange = useCallback(
     (newEditorMode: EditorMode) => {
+      reportInteraction('grafana_sql_editor_changed', {
```

Comment:
```suggestion
      reportInteraction('grafana_sql_editor_mode_changed', {
```

maybe we should add the `mode` word to the interaction-name, WDYT?